tables/create_archive_manager.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE archive_manager
+    (   first_name VARCHAR(32),
+        last_name VARCHAR(32), 
+        handle VARCHAR(16), 
+        email  VARCHAR(255), 
+        access_level VARCHAR(32), 
```

Comment:
We need another datatype here. This column needs to only select one of the access levels predefined. Think of this as similar to drop-down list. Find the appropriate datatype from MySQL documentation.

Reply:
I added a constraint making sure that it only accepts AR-L1,AR-L2, AR-L3, etc.,
ALTER TABLE archive_manager
    -> ADD CONSTRAINT chk_lvl CHECK(access_level in ('AR-L1', 'AR-L2', 'AR-L3','AR-L4'));